src/editors/containers/TextEditor/components/ImageSettingsModal/AltTextControls.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form } from '@edx/paragon';
+
+import * as hooks from './hooks';
+
+/**
+ * Wrapper for alt-text input and isDecorative checkbox control
+ * @param {bool} isDecorative - is the image decorative?
+ * @param {func} setIsDecorative - handle isDecorative change event
+ * @param {func} setValue - update alt-text value
+ * @param {string} value - current alt-text value
+ */
+export const AltTextControls = ({
+  isDecorative,
+  setIsDecorative,
+  setValue,
+  value,
+}) => (
+  <Form.Group className="mt-4.5">
+    <Form.Label as="h4">Accessibility</Form.Label>
+    <Form.Control
+      className="mt-4.5"
+      type="input"
+      value={value}
+      disabled={isDecorative}
+      onChange={hooks.onInputChange(setValue)}
+      floatingLabel="Alt Text"
+    />
```

Comment:
So my question here, better phrased is this:

Is disabling the save button a sufficient amount of feedback to give to the user that they need to either set the image to be decorative or add alt text? To me, It doesn't, it just looks like something is brokem. I think it would be prudent to add some feedback to the user that they need to fill in either of these. The `Form.Control.Feedback` component in paragon might be helpful to do that. If this is future work, I'm more than happy to write a ticket, but this feels important enough to include in this.

Reply:
For right now, I would call that a new ticket, as the behavior is not spelled out in the design, and where/how we would want that feedback to be indicated is a design decision :-)